### PR TITLE
Separate align and alignment, and add hole as an alias for void.

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: api/shape/alignment.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import Point from './Point.js';
```
Comment (BarbourSmith) [inline api/shape/alignment.js:1]: I like the name change for this file

File: api/shape/void.js [hunk 2]
```diff
@@ -3,7 +3,9 @@ import { hasTypeGhost, hasTypeVoid } from '@jsxcad/geometry';
 import Shape from './Shape.js';
 
 export const voidFn = Shape.registerMethod(
-  'void',
+  ['void', 'hole'],
```
Comment (BarbourSmith) [inline api/shape/void.js:6]: Hole is nice

File: nb/api/align.md [hunk 3]
```diff
@@ -25,40 +25,40 @@ Box(10, 10, 1).view().note('Box(10, 10, 1)');
...
+Box(10, 10, 1).align('x>').and(origin).view().md("align('x>')");
 ```
 
 ![Image](align.md.$5.png)
 
 align('x<')
 
 ```JavaScript
-Box(10, 10, 1).by(align('x<')).and(origin).view().md("align('x<')");
+Box(10, 10, 1).align('x<').and(origin).view().md("align('x<')");
```
Comment (BarbourSmith) [inline nb/api/align.md:47]: This is way easier to read I think

**Review by BarbourSmith — APPROVED** [review]
Looks good to me!

File: api/shape/void.js (hunk 2 above)
Comment (pentacular) [reply api/shape/void.js:6]: Yeah, the critical problem is the inability to write void() :)

File: nb/api/align.md (hunk 3 above)
Comment (pentacular) [reply nb/api/align.md:47]: Yes, I think so -- using alignment() should be an unusual case.

**Pull request merged**